src/common/Utilities.cpp

```diff
@@ -49,6 +49,19 @@ namespace fs = boost::filesystem;
 
 namespace scrimmage {
 
+std::string exec_command(const char* cmd) {
```

Comment:
Since we aren't actually using this command in scrimmage, it seems more appropriate to put it in scrimmage_ros

Reply:
I agree. I'm going to move this function out of scrimmage core.